pyasic/config/mining.py

```diff
@@ -152,14 +152,14 @@ def as_epic(self) -> str:
 class VOptAlgo(MinerConfigValue):
     mode: str = field(init=False, default="voltage_optimizer")
 
-    def as_epic(self) -> str:
```

Comment:
This seems wrong, should still reference self...

Could make it a static method, but not much point really.